src/NzbDrone.Core/Parser/QualityParser.cs

```diff
@@ -37,7 +37,7 @@ public class QualityParser
         private static readonly Regex RealRegex = new Regex(@"\b(?<real>)real\b",
                                                                 RegexOptions.Compiled | RegexOptions.IgnoreCase);
 
-        private static readonly Regex ResolutionRegex = new Regex(@"\b(?:(?<_480p>480p|640x480|848x480)|(?<_576p>576p)|(?<_720p>720p|1280x720)|(?<_1080p>1080p|1920x1080))\b",
+        private static readonly Regex ResolutionRegex = new Regex(@"(?:(?<_480p>480p|640x480|848x480|bdrip)|(?<_576p>576p)|(?<_720p>720p|1280x720)|(?<_1080p>1080p|1920x1080))\b",
```

Comment:
it's because before the source regex wasn't matching the `bd` in `bdrip`

I think it's more accurate now, since the source is actually Blu-Ray.